nibabel/nicom/dicomwrappers.py

```diff
@@ -21,6 +21,14 @@
 from ..openers import ImageOpener
 from ..onetime import setattr_on_read as one_time
 
+try:
```

Comment:
Yes, this sounds like a good idea.  Would it go in the `nicom` package, or somewhere else?